extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
Yeah--sorry about this.  I had meant to remove all the js styling, but I guess this one got through.  I have removed it in head, but for uses of 1.1.0, !important is the right workaround I think.